### PR TITLE
Tons of fixes.

### LINKED ISSUE #21: this nonsense needs a better design
http://screencast.com/t/SsKfSjcTd0

### LINKED ISSUE #38: Using ' ' instead of " " across lavender website
This was discussed Friday, but I know that this is an issue I caused (whoops). Fluid use of ' ' instead of " " across the site. Or vice-versa. Anyone have any comments on which to use? It's easy to fix once decision is made.

### LINKED ISSUE #40: DRY Footer

### LINKED ISSUE #41: Conditional comments shorthand should be readable or consistent
In the language guide, the shorthand for conditional comments appears to be IT in the narrative and IE in the code. Also, the link "this if you are not familiar with conditional comments" maybe should just be on the word "this"

### LINKED ISSUE #43: Background image should live on our server
main.css line 7
background-image: url('https://ununsplash.imgix.net/photo-1415045384817-2f9cf7f2ed79?q=75&fm=jpg&s=a0d8ebf44a449c7855ca4128e914a7c5');

### PR DESCRIPTION
**Pull request by sugarPhlox**
fixes #43 - Background image should live on our server
fixes #41 - Conditional comments shorthand should be readable or consistent
fixes #40 - DRY footer
fixes #38 - Using ' ' instead of " " across lavender website
fixes #21 - this nonsense needs a better design

http://screencast.com/t/8zE3WaITozNy - about page + new footer
http://screencast.com/t/aOqFZsHamts - organized nonsense

(Also, I don't understand why I can merge 50% of the time into development, and the other 50% I get issues. And I do the same thing.)

### CODE REVIEW

**Comment by avophile** [comment]
The footer is pretty! I am not worthy to give cake but consider this a crumb!

File: views/guide.lavender [hunk 1]
```diff
@@ -91,7 +91,7 @@ block content
             | Add the lavender submodule with 
             code git submodule add git@github.com:golavender/lavender.git (folder to 
             | clone to) and include it with 
-            code require "(where you put lavender)/src/Lavender/lavender.php   
+            code require '(where you put lavender)/src/Lavender/lavender.php
```
Comment (mowchan) [inline views/guide.lavender]: Think we need a closing ' at the end of this line.

```
'(where you put lavender)/src/Lavender/lavender.php'
```

File: views/guide.lavender [hunk 2]
```diff
@@ -138,8 +138,8 @@ block content
             | becomes
             pre
               code.
-                &lt;section id="some_id" class="foo bar" attribute1="foo" 
-                attribute2="bar"&gt;&lt;/section&gt;
+                &lt;section id='some_id' class='foo bar' attribute1='foo' 
+                attribute2='bar'&gt;&lt;/section&gt;
```
Comment (mowchan) [inline views/guide.lavender:142]: Is there a reason why we're using &lt &gt instead of just the < / > characters?

Comment (sugarPhlox) [inline views/guide.lavender:142]: I had problems with this trying to execute itself, even when inside the code tag. I think this is a problem carrying over from the language itself?

Comment (mowchan) [inline views/guide.lavender:142]: Word, I figured that might be the case.

**Comment by mowchan** [comment]
:cake: Usually when you are unable to automatically merge, it means that one (or more) of the files you worked on has changed since you started work on this branch. I would bet that something else has been merged into master since you started working on this branch.

**Comment by zlincoln** [comment]
:cake: Yea, if you pull development into this branch and push the branch to github, you'll then have the ability to merge.

**Pull request merged**